Platform/de.lproj/Localizable.strings

```diff
@@ -91,10 +91,10 @@
 "Are you sure you want to permanently delete this disk image?" = "Möchten Sie dieses Abbild wirklich dauerhaft löschen?";
 
 /* No comment provided by engineer. */
-"Are you sure you want to reset this VM? Any unsaved changes will be lost." = "Möchten Sie wirklich einen Reset ausführen? Ungespeicherte Änderungen gehen verloren.";
+"Are you sure you want to reset this VM? Any unsaved changes will be lost." = "Möchten Sie wirklich einen Reset der VM ausführen? Alle ungespeicherten Änderungen gehen verloren.";
```

Comment:
〽️ Adding the word "dabei" enriches the translation slightly and makes it more specific. I understand that this may well be a personal style preference, hence accept or reject this suggestion as you please.

```suggestion
"Are you sure you want to reset this VM? Any unsaved changes will be lost." = "Möchten Sie wirklich einen Reset der VM ausführen? Alle ungespeicherten Änderungen gehen dabei verloren.";
```